python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -335,6 +385,62 @@ def _get_run_tags(self, run_id):
 
         return Tags(arn, cluster, cpu, memory)
 
+    def _run_task(self, run_task_kwargs, capacity_failure_strategy: dict, fallback_capacity_provider_strategy: list):
```

Comment:
`dict` and `list` don't work in the versions of Python that dagster needs to support - I think you want Dict and List